code/addons/vitest/src/plugin/test-utils.ts

```diff
@@ -41,23 +47,7 @@ export const testStory = (
       meta: TaskMeta & { storyId: string; hasPlayFunction: boolean };
     };
     _task.meta.storyId = Story.id;
```

Comment:
Maybe we should make the args, parameters and id also available on the vitest context:

See context extension here:
https://vitest.dev/guide/test-context.html#beforeeach-and-aftereach

```tsx
async (context: TaskContext) => {
  // extend context
  context.story = { args, parameters, id, tags }
})
```

Or for another PR

Reply:
Another PR